docs/design/processing_trace_data/records_service.md

```diff
@@ -0,0 +1,214 @@
+# Records service
+
+The Records object holds time-series data such as message flow in a table as shown below.
+
+| Start timestamp | ... | End timestamp |
```

Comment:
I think that `start timestamp` and `end timestamp` are better because large character does not have meaning here.